local-deploy/0.variables.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Chain configuration
+CHAIN_ID=203
+NETWORK_ID=203
+NUM_CNS=1
+DOCKER_IMAGE=kaiachain/kaia:latest
+
+# Node configuration (uncomment if needed)
+# PRIVATE_KEY=492cf9f1e9574e7f9f3e914dcf18117280cf238776ca715da47996ac1b36b98b
```

Comment:
I know it's a test key, but what about removing the specific private key contents?